src/index.js

```diff
@@ -1,93 +1,100 @@
 const app = express();
 
 app.use(cors());
 
 // Normal express config defaults
-app.use(require("morgan")("dev"));
+app.use(require('morgan')('dev'));
+
 app.use(bodyParser.urlencoded({ extended: false }));
 app.use(bodyParser.json());
 
-app.use(require("method-override")());
-app.use(express.static(__dirname + "/public"));
+app.use(require('method-override')());
+
+app.use(express.static(`${__dirname}/public`));
+
+app.get('/', (req, res) => res.status(200).json({
+  status: 200,
+  message: 'Welcome To Bare Foot Nomad',
+}));
 
 app.use(
-    session({
-        secret: "authorshaven",
-        cookie: { maxAge: 60000 },
-        resave: false,
-        saveUninitialized: false
-    })
+  session({
+    secret: 'authorshaven',
+    cookie: { maxAge: 60000 },
+    resave: false,
+    saveUninitialized: false
+  })
 );
 
 if (!isProduction) {
-    app.use(errorhandler());
+  app.use(errorhandler());
 }
 
 if (isProduction) {
-    mongoose.connect(process.env.MONGODB_URI);
+  mongoose.connect(process.env.MONGODB_URI);
 } else {
-    mongoose.connect("mongodb://localhost/conduit");
-    mongoose.set("debug", true);
+  mongoose.connect('mongodb://localhost/conduit');
+  mongoose.set('debug', true);
 }
 
-require("./models/User");
+require('./models/User');
 
-app.use(require("./routes"));
+app.use(require('./routes'));
 
-/// catch 404 and forward to error handler
-app.use(function(req, res, next) {
-    const err = new Error("Not Found");
-    err.status = 404;
-    next(err);
+// / catch 404 and forward to error handler
+app.use((req, res, next) => {
+  const err = new Error('Not Found');
+  err.status = 404;
+  next(err);
 });
 
-/// error handlers
+// / error handlers
 
 // development error handler
 // will print stacktrace
 if (!isProduction) {
-    app.use(function(err, req, res, next) {
-        console.log(err.stack);
+  app.use((err, req, res, next) => {
+    console.log(err.stack);
 
-        res.status(err.status || 500);
+    res.status(err.status || 500);
 
-        res.json({
-            errors: {
-                message: err.message,
-                error: err
-            }
-        });
+    res.json({
+      errors: {
+        message: err.message,
+        error: err
+      }
     });
+  });
 }
 
 // production error handler
 // no stacktraces leaked to user
-app.use(function(err, req, res, next) {
-    res.status(err.status || 500);
-    res.json({
-        errors: {
-            message: err.message,
-            error: {}
-        }
-    });
+app.use((err, req, res, next) => {
+  res.status(err.status || 500);
+  res.json({
+    errors: {
+      message: err.message,
+      error: {}
+    }
+  });
 });
 
 // finally, let's start our server...
-const server = app.listen(process.env.PORT || 3000, function() {
-    console.log("Listening on port " + server.address().port);
+const server = app.listen(process.env.PORT || 3000, () => {
+  console.log('Listening on port ' + server.address().port);
 });
```

Comment:
I see that you had to lint this file, awesome! However, you did not convert the `require` to `import` statements.

Reply:
I actually just did what was within the scope of the chore which was to include the npm scripts as well as set the home routes.